source/build/GDK/TestClientApp.vcxproj.ejs

```diff
@@ -173,7 +173,7 @@ for (var file in projectFiles.TestClientAppFiles) {
     <ClCompile Include="$(PlayFabTestDir)\TestApp\GDK\DeviceResources.cpp" />
     <ClCompile Include="$(PlayFabTestDir)\TestApp\GDK\TestAppGDK.cpp" /><%
 for(var featureGroupName in SDKFeatureGroups) { 
-    if (featureGroupName === "Groups") {  %>
+    if ([ "Groups", "Data" ].includes(featureGroupName)) {  %>
```

Comment:
Would it make sense to put this collection of "Implemented test groups" into a separate config variable somewhere? Mostly for readability.

Reply:
I think its fine to leave it as is since this is a temporary thing anyhow

Reply:
We definitely could. In theory, this won't live long though, since we won't need this check once the other few feature groups are added. Maybe if there'd be some reason to keep around a list in case we need to continually disable/reenable tests in the future? I can't see what that'd be though.

Reply:
Fair point. Agreed on this being temporary and not worth complicating in the short term. Resolving.